packages/standards/biome.jsonc

```diff
@@ -33,17 +33,22 @@
         // be the case. https://biomejs.dev/linter/domains/#project-rules
         "noFloatingPromises": "error",
 
+        "noSecrets": "error",
         "noShadow": "error",
       },
       "performance": {
         "noBarrelFile": "error",
       },
       "style": {
         "noParameterAssign": "error",
+        "useConsistentArrayType": "error",
+        "useDefaultParameterLast": "error",
         "useDefaultSwitchClause": "error",
+        "useThrowNewError": "error",
```

Comment:
This was enabled in our default ESLint config via [unicorn/throw-new-error](https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/throw-new-error.md).